tests/unit/test_masks.py

```diff
@@ -138,6 +142,24 @@ def it_knows_its_mask_slide(
 
         assert binary_mask == expected_mask
 
+    def it_knows_its_mask_slide_custom(self, request, tmpdir):
+        slide, _ = base_test_slide(tmpdir, PILIMG.RGBA_COLOR_500X500_155_249_240)
+        custom_filters = [image_filters.RgbToGrayscale()]
+
+        set_custom_filters_ = method_mock(
+            request, _CustomFiltersComposition, "set_custom_filters"
+        )
+        custom_filter_call_ = method_mock(
+            request, image_filters.RgbToGrayscale, "__call__"
+        )
+
+        tissue_mask = TissueMask(*custom_filters)
+        _ = tissue_mask(slide)
+
+        set_custom_filters_.assert_called_once()
```

Comment:
I'm not a huge fan of `assert_called_once`. The reason is that, yes, you're asserting that a proper method has been called, but the interesting info would be also know with which parameters? I'd suggest to switch to `assert_called_once_with` and check the calling args 😉

Reply:
I try to remove all of the things. However, I sometimes have no access to the "self" parameter. In this case, I try to check only the remaining arguments.

Reply:
Sorry I do not understand this point. Could you elaborate? Thx